### ISSUE TITLE
Export FQN

### ISSUE THREAD

**Issue by cedricziel** [NamelessCoder/fluid-documentation-generator#2]
Can the FQCN of the class name be exported as well?

**Comment by NamelessCoder** [comment]
Yes, but it requires the PHP namespace prefix to be set in metadata.json (see README) - and I'm not sure that it currently does export the induced class name. There's furthermore the complication that even with the PHP namespace prefix configured, the expected class name can only be inferred, not actually checked, which may yield wrong values for multi-PHP-namespace Fluid namespaces.

Unfortunately the XSD format doesn't immediately allow setting this either. At least not in any attributes known to me currently...
